### PR TITLE
repository name != pypi name follow-up

### LINKED ISSUE #170: repository name != pypi name is a problem
We ran into this issue when doing the first release in [packit](https://github.com/packit-service/packit). The name in PyPI is packitos and sadly release-bot checked against PyPI repository packit, not packitos.

TODO:
* [ ] add a new option to config file to set PyPI project name:
  ```
  pypi_project: packitos
  ```
* [ ] optionally, try to parse the project name from setup.cfg (this is a stretch goal)

### PR DESCRIPTION
**Pull request by rpitonak**
This is follow-up PR for #182 

Fixes #170

### CODE REVIEW

File: release_bot/configuration.py [hunk 1]
```diff
@@ -121,8 +124,8 @@ def load_release_conf(self, conf):
         parsed_conf = yaml.safe_load(conf) or {}
         # If pypi option is not specified in release-conf.yaml,
         # it defaults to true.
-        if parsed_conf.get('pypi') is None:
-            parsed_conf['pypi'] = True
+        parsed_conf['pypi'] = parsed_conf.get('pypi') or True
```
Comment (jpopelka) [inline release_bot/configuration.py]: either
```python
  parsed_conf['pypi'] = parsed_conf.get('pypi', True)
```
or even better with [setdefault()](https://www.tutorialspoint.com/python/dictionary_setdefault.htm)
```python
  parsed_conf.setdefault('pypi', True)
```

**Comment by rpitonak** [comment]
`TestLoadReleaseConf.test_different_pypi_name` failling in [CentOS CI](https://ci.centos.org/job/release-bot-pr/53/console), but passing in [Travis](https://travis-ci.org/user-cont/release-bot/jobs/517658035) and also locally. Can someone else also verify it? then we can merge I guess.

**Comment by Aniket-Pradhan** [comment]
[Working](https://paste.fedoraproject.org/paste/ynQSjHcPrgYdcyQ8AOPL2w) for me. :smile:

**Comment by TomasTomecek** [comment]
Trying locally and I have found more broken tests.

**Comment by TomasTomecek** [comment]
I pushed a bunch fixes which should make like 15 tests pass.

**Comment by TomasTomecek** [comment]
@jpopelka +1 <3

Let's merge since CI is passing, I'll send remaining test fixes as a new PR.

**Comment by TomasTomecek** [comment]
One pull request, 4 different collaborators: @Elias999 @rpitonak @TomasTomecek @jpopelka 

Wooooh! We rock!

**Pull request merged**